<!--     -->
<launch>
    <param name="follow_offset" value="0.4"/>
    <param name="follow_advance" value="1.0"/>
    
    <param name = "orthog_distance_threshold" type="double" value="0.05"/>
    <param name = "min_points_per_line" type="int" value="4"/>
    <param name = "maximum_range" type="double" value="3.25"/>
    
    <param name="lines_topic" value="/extracted_lines"/>
    <param name="vis_lines_topic" value="/visualization_lines"/>
    <param name="vis_scanpoints_topic" value="/visualization_scanpoints"/>

    <node pkg="stage_ros" type="stageros" name="stageros" args="$(find comp4766_a3_world)/world/autolab.world"/>
    <!-- <node pkg="rviz" type="rviz" name="rviz"/>  -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find comp4766_a3_mod)/rviz.rviz"/>
</launch>
